src/nvim/globals.h

```diff
@@ -847,8 +847,8 @@ EXTERN int* (*iconv_errno)(void);
 EXTERN int State INIT(= NORMAL);        /* This is the current state of the
                                          * command interpreter. */
 
-EXTERN int finish_op INIT(= FALSE);     /* TRUE while an operator is pending */
-EXTERN long opcount INIT(= 0);          /* count for pending operator */
+EXTERN int finish_op INIT(= false);     // TRUE while an operator is pending
```

Comment:
as you had to change this line anyway due to linting, you can just as well update it completely :) (`bool` type and `true` in comment)